packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -60,19 +69,33 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownPropsAndState, DropdownVa
 
   clearIndicator: getIndicatorStyles,
 
-  container: ({ props: p, variables: v }): ICSSInJSStyle => ({
+  container: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => ({
     display: 'flex',
     flexWrap: 'wrap',
     position: 'relative',
     borderStyle: 'solid',
-    borderColor: 'transparent',
+    borderColor: v.borderColor,
     outline: 0,
     width: getWidth(p, v),
-    borderWidth: v.borderWidth,
-    borderRadius: v.borderRadius,
+    borderWidth: p.search ? `0 0 ${v.searchBorderBottomWidth} 0` : v.borderWidth,
+    borderRadius: getContainerBorderRadius(p, v),
     color: v.color,
     backgroundColor: v.backgroundColor,
-    ...(p.focused && { borderBottomColor: v.borderColorFocus }),
+    ':hover': {
+      backgroundColor: v.backgroundColorHover,
+      [`& .${Dropdown.slotClassNames.triggerButton}`]: {
+        // reset all styles
+      },
+    },
+    ...(p.focused && {
```

Comment:
Found out what is causing the issue with the trigger button, but no idea why :( It's the styles applied when the container of the Dropdown is focused:
```
  ...(p.focused && {
     ...(p.search && { borderBottomColor: v.borderColorFocus }),
     ...(!p.search &&
        !p.open &&
        getBorderFocusStyles({
          siteVariables,
          isFromKeyboard: true,
        })[':focus']),
    }),
```
@Bugaa92 am I using this function as intended? In the Dropdown container, we are using the `focused` prop as an indicator whether it is focused. FYI @silviuavram

Reply:
Maybe the reason for breaking this is the added pseudo elements: `::before` and `::after`

Reply:
I fixed this, by adding isFromKeyboard in the Dropdown state. Let's see if this will turn around again as an issue.